src/utils/connectToDatabase.js

```diff
@@ -0,0 +1,54 @@
+const mongodb = require('mongodb');
```

Comment:
Yeah I think this file is the perfect thing to make a small npm package sometime this week or next